src/test/kotlin/org/rust/cargo/project/RustProjectSettingsServiceTest.kt

```diff
@@ -31,7 +31,7 @@ class RustProjectSettingsServiceTest : LightPlatformTestCase() {
               <option name="toolchainHomeDirectory" value="/" />
               <option name="useCargoCheckAnnotator" value="true" />
               <option name="useCargoCheckForBuild" value="false" />
-              <option name="useOfflineForCargoCheck" value="true" />
+              <option name="useOffline" value="true" />
```

Comment:
@mchernyavsky I assume renaming option will cause its setting to be lost on update?
Is this OK/expected or should we try to stay compatible with previous settings?

Reply:
I think it's OK since this option is not widely used.
And before this commit, `useOfflineForCargoCheck` had no effect on `cargo check` :)